bin/MadGraph5_aMCatNLO/cards/production/13TeV/HToZATo2L2B_ggfusion_b-associatedproduction/example_cards/HToZATo2L2B_500p00_300p00_1p50_ggH_TuneCP5_13TeV_pythia8/HToZATo2L2B_500p00_300p00_1p50_ggH_TuneCP5_13TeV_pythia8_run_card.dat

```diff
@@ -118,31 +136,64 @@ $DEFAULT_PDF_SETS = lhaid ! if pdlabel=lhapdf, this is the lhapdf number
 #*********************************************************************
 # Maximum and minimum absolute rapidity (for max, -1 means no cut)   *
 #*********************************************************************
+  5.0  = etaj  ! max rap for the jets 
   -1.0	= etab ! max rap for the b
   -1.0	= etal ! max rap for the charged leptons 
+  -1.0  = etal    ! max rap for the photons
```

Comment:
etal as well, though it won't matter since the same cuts are applied, I concern that possibly other cuts might have been defined twice somewhere

Reply:
Did you somehow mix up the NLO and LO run cards? I would suggest to start things fresh

Reply:
yes, it seems so :( , I have clean up the run_card. 
Please have a look.